app/models/upload.rb

```diff
@@ -14,6 +14,7 @@ class Upload < ApplicationRecord
   validate :valid_url, if: proc { |upload| upload.url.present? }
   scope :active, -> { where(status: %w[active processed]) }
   scope :archived, -> { where(status: 'archived') }
+  default_scope { order(created_at: :desc) }
```

Comment:
I'm a little wary of adding a default scope that sets the order on uploads. There are possible unintended side effects if other features are relying on the order being different.